### PR TITLE
VE-focuswidget: WikiaFocusWidget integration

### PR DESCRIPTION
**Pull request by BladeBronson**
Building on the work of the other viewpagetarget pull request, this re-introduces the focus widget.
- Surface receives reference to target
- Actions toolbar config is using static config for Wikia overrides
- Sequencing to allow for Venus animation delays
- Importing stylesheets
- Fixed paths to icon directories in the icon mixin

### CODE REVIEW

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTransclusionDialog.scss [hunk 1]
```diff
@@ -0,0 +1,105 @@
+.ve-ui-mwTemplateDialog {
```
Comment (inez) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTransclusionDialog.scss]: @BladeBronson Is importing this file your intentional action?

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTemplateInsertDialog.scss [hunk 2]
```diff
@@ -0,0 +1,77 @@
+@import "skins/shared/color";
```
Comment (inez) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTemplateInsertDialog.scss]: @BladeBronson Is importing this file your intentional action?

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSourceModeDialog.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
+@import "skins/shared/color";
```
Comment (inez) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSourceModeDialog.scss]: @BladeBronson Is importing this file your intentional action?

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSingleMediaInsertDialog.scss [hunk 4]
```diff
@@ -0,0 +1,195 @@
+@import "skins/shared/mixins/transition";
```
Comment (inez) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSingleMediaInsertDialog.scss]: @BladeBronson Is importing this file your intentional action?

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaPreferenceDialog.scss [hunk 5]
```diff
@@ -0,0 +1,51 @@
+@import "skins/shared/color";
```
Comment (inez) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaPreferenceDialog.scss]: @BladeBronson Is importing this file your intentional action?

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaOrientationDialog.scss [hunk 6]
```diff
@@ -0,0 +1,45 @@
+@import "skins/shared/color";
```
Comment (inez) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaOrientationDialog.scss]: @BladeBronson Is importing this file your intentional action?

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaMediaPreviewWidget.scss [hunk 7]
```diff
@@ -0,0 +1,50 @@
+@import "skins/oasis/css/core/layout";
```
Comment (inez) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaMediaPreviewWidget.scss]: @BladeBronson Is importing this file your intentional action?

**Comment by inez** [comment]
Can you please target it against ve-sprint-33-upstream-sync? And also merge changes from that branch to tests can run correctly.

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTransclusionDialog.scss (hunk 1 above)
Comment (BladeBronson) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTransclusionDialog.scss]: Yes.

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTemplateInsertDialog.scss (hunk 2 above)
Comment (BladeBronson) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaTemplateInsertDialog.scss]: Yes.

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSourceModeDialog.scss (hunk 3 above)
Comment (BladeBronson) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSourceModeDialog.scss]: Yes.

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSingleMediaInsertDialog.scss (hunk 4 above)
Comment (BladeBronson) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSingleMediaInsertDialog.scss]: Yes.

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaPreferenceDialog.scss (hunk 5 above)
Comment (BladeBronson) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaPreferenceDialog.scss]: Yes.

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaOrientationDialog.scss (hunk 6 above)
Comment (BladeBronson) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaOrientationDialog.scss]: Yes.

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaMediaPreviewWidget.scss (hunk 7 above)
Comment (BladeBronson) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaMediaPreviewWidget.scss]: Whoa, how did this get in there!!??

**Pull request closed without merging**